src/Rector/Deprecation/ConstantToClassConstantRector.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use PhpParser\Node;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * Replaces deprecated constant with class constant.
+ *
+ * What is covered:
+ * - Replacement with a use statement.
+ */
+class ConstantToClassConstantRector extends AbstractRector implements ConfigurableRectorInterface
+{
+    /**
+     * The deprecated constant.
+     *
+     * @var string
+     */
+    protected $deprecatedConstant;
+
+    /**
+     * The replacement fully qualified class name.
+     *
+     * @var string
+     */
+    protected $constantFullyQualifiedClassName;
+
+    /**
+     * The replacement constant.
+     *
+     * @var string
+     */
+    protected $constant;
+
+    const DEPRECATED_CONSTANT = 'deprecated_constant';
+    const CONSTANT_FULLY_QUALIFIED_CLASS_NAME = 'constant_fully_qualified_class_name';
+    const CONSTANT = 'constant';
+
+    /**
+     * @param array $configuration
+     */
+    public function configure(array $configuration): void
+    {
+        $this->deprecatedConstant = $configuration[static::DEPRECATED_CONSTANT];
+        $this->constantFullyQualifiedClassName = $configuration[static::CONSTANT_FULLY_QUALIFIED_CLASS_NAME];
+        $this->constant = $configuration[static::CONSTANT];
+    }
```

Comment:
We should add validation here, which may help debug. Apply `?? ''` to each and then check if any property is `=== ''` and throw an exception that it must be properly configured.